README.md

```diff
@@ -202,6 +202,77 @@ If query text is present, the translation of query text based on input language
 
 If the query text is absent and audio url is present, then the audio url is downloaded and converted into text based on the input language. Once speech to text conversion in input language is finished, the same process mentioned above happens. One difference is that by default, the paraphrased answer is converted to voice irrespective of the output_format since the input format is voice.
 
+### `POST /v1/chat`
+
+#### API Function
+API is used to generate activity/story based on user query and translation of text/audio from one language to another language in text/audio format. To achieve the same, Bhashini has been integrated. OCI object storage has been used to store translated audio files when audio is chosen as target output format.
```

Comment:
With the new packaging implementation, we are enabling bhashini, google, dhruva for language

OCI object storage - can be now S3 , Azure etc

Can we capture this information for both 

`/v1/query and /v1/chat
`